src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -81,6 +97,10 @@ public class PatchMapDrawer : Control
     public override void _Ready()
     {
         nodeScene = GD.Load<PackedScene>("res://src/microbe_stage/editor/PatchMapNode.tscn");
+        connections = new Dictionary<Tuple<int, int>, Tuple<Vector2, Vector2, Vector2>>();
```

Comment:
These tuples are too much for readability, they should be changed to be class types. For the left side Int2 might be good.